python2.txt

```diff
@@ -29,7 +29,7 @@ Description = Un Linguaggio di programmazione dinamico e potente.
 Description = Een opmerkelijk krachtige dynamische programmeertaal.
 
 [Section.0415]
-Description = Potęży i dynamiczny język programowania.
+Description = Niezwykle potężny dynamiczny język programowania.
```

Comment:
```suggestion
Description = Niezwykle potężny, dynamiczny język programowania.
```

Reply:
I suggest not writing commas in addition because of Polish grammar. Here's the explanation (in Polish): https://sjp.pwn.pl/poradnia/haslo/przydawki-a-przecinek;6290.html

Reply:
OK